guides/MoodleIntegrationGuide.md

```diff
@@ -38,10 +51,14 @@ To successfully complete this installation the system administrator will require
 * privileges to unzip and create files on the Moodle file system.
 
 #### System prerequisites for integration
-•  openEQUELLA opens source and Moodle version 2.6, 2.7 or 2.8 must be installed and running. Further information on installing openEQUELLA is provided in the openEQUELLA Installation and Administration Guide.
-•  Ensure the time settings of the servers are set correctly, including DST, for the relevant time zone.
-•  To access the openEQUELLA Dashboard page at least one course must have been created in Moodle. Refer to the Moodle documentation for more information. 
 
+* openEQUELLA opens source and Moodle version 2.6, 2.7 or 2.8 must be installed and running. Further
+  information on installing openEQUELLA is provided in the openEQUELLA Installation and
+  Administration Guide.
+* Ensure the time settings of the servers are set correctly, including DST, for the relevant time
+  zone.
+* To access the openEQUELLA Dashboard page at least one course must have been created in Moodle.
+  Refer to the Moodle documentation for more information.
```

Comment:
Formatting issue only, still had actual bullets rather than markdown list syntax.

(Note though the references to old Moodle versions.)